dmscripts/generate_agreement_signature_pages.py

```diff
@@ -27,6 +27,26 @@ def render_html_for_successful_suppliers(rows, framework, template_dir, output_d
     shutil.copyfile(template_css_path, os.path.join(output_dir, '{}-signature-page.css'.format(framework)))
 
 
+def render_html_for_suppliers_awaiting_countersignature(rows, framework, template_dir, output_dir):
+    template_path = os.path.join(template_dir, '{}-counterpart-signature-page.html'.format(framework))
+    template_css_path = os.path.join(template_dir, '{}-signature-page.css'.format(framework))
+    countersignature_img_path = os.path.join(template_dir, '{}-countersignature.png'.format(framework))
+    for data in rows:
+        if data['on_framework'] is False or not data['countersigned_at'] or data['countersigned_path']:
+            print("SKIPPING {}: on_f={} at={} path={}".format(
+                data['supplier_id'],
+                data['on_framework'],
+                data['countersigned_at'],
+                data['countersigned_path'])
+            )
```

Comment:
These aren't errors - just agreements that have already been generated, but definitely better to use a proper logger - I'll fix it.